javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,15 @@
+var fullName = 'Angela Jia Nan';
+console.log('value of fullName:', fullName);
+var isCool = true;
+console.log('value of isCool:', isCool);
+var totalPets = 1;
+console.log('value of totalPets:', totalPets);
+console.log('typeof fullName:', typeof fullName);
+console.log('typeof isCool:', typeof true);
```

Comment:
for these, you should be using the variable, not the value contained in the variable